js/webxr-sample-app.js

```diff
@@ -146,12 +147,32 @@ export class WebXRSampleApp {
     }
 
     this.onInitRenderer();
+    const gl = this.gl;
 
     this.scene.inputRenderer.useProfileControllerMeshes(session);
 
-    session.updateRenderState({
-      baseLayer: this.onCreateXRLayer(session)
-    });
+    if (session.isImmersive && this.options.useMultiview) {
+      this.renderer = new Renderer(gl, true, true);
+      this.scene.setRenderer(this.renderer);
+
+      let xrGLFactory = new XRWebGLBinding(session, gl);
```

Comment:
Not sure if this is used by any of the samples; but also seems to be layer dependent?

Reply:
yes